batch/aiml-workloads/kubernetes-manifests/persistent-volume.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Awesome job with this tutorial @ahrarmonsur!! 🔥 
I saw 2 region tags were being added, can you also add region tags for the other 4 yaml files?

Reply:
Added region tags to all YAML files!